scripts/protocgen.sh

```diff
@@ -13,15 +13,15 @@ protoc_gen_gocosmos() {
 
 protoc_gen_doc() {
   go get -u github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc 2>/dev/null
-} 
+}
 
 protoc_gen_gocosmos
 protoc_gen_doc
 
 proto_dirs=$(find ./proto -path -prune -o -name '*.proto' -print0 | xargs -0 -n1 dirname | sort | uniq)
 # TODO: migrate to `buf build`
 for dir in $proto_dirs; do
-  buf protoc \
+  buf alpha protoc \
```

Comment:
should we migrate to `buf build` instead? `buf alpha protoc` will be deprecated soon

Reply:
Yes. I have not looked into what is needed for `buf build` to work. 
This PR is a temporary fix, for `make proto-gen` to work again.